vaults/sources/Vault.move

```diff
@@ -17,18 +17,27 @@ module titusvaults::Vault {
 
     struct CurrentRound has key {
         intial_time: u64,
-        round: u64,
+        round: u64
     }
 
     struct Vault <phantom VaultT, phantom AssetT> has key {
         coin_store: Coin<AssetT>,
         creation_time: u64,
         creation_round: u64,
+        total_shares: u64
```

Comment:
are this total shares in a round or total shares in all rounds combined?

Reply:
all rounds combined